packages/program-boilerplate/tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+  "include": [
+    "src/**/*.ts"
+  ],
+  "compilerOptions": {
+    "outDir": "dist",
+    "module": "commonjs",
+    "target": "es2015",
+    "noImplicitAny": false,
+    "jsx": "react"
```

Comment:
Are we using react in boilerplate?

Reply:
Nope, I just copied this from the keen query server... which also isn't using react haha. Not sure why that's in there